modules/ROOT/pages/sid-reference.adoc

```diff
@@ -0,0 +1,244 @@
+= Sensitive Information Detection
+
+Sensitive Information Detection (SID) allows you to identify where you have potential leakage of sensitive information.
+
+These are some common scenarios where implementing SID can help you keep sensitive data more secure: 
+
+* Developers use sensitive information when they build apps. If developers aren’t aware of what is sensitive information and what is not, they can inadvertently cause leakage by exposing the sensitive information.
+* Developers build test APIs, or add testing parameters to existing APIs, that return sensitive information. These APIs or parameters were not documented, and the developer forgets to remove the sensitive information from the API.  
+* Data is received from partners that may contain sensitive information, which should be cleansed.
+
```

Comment:
There's something funky about the point-of-view or tense of these three bullets. Can you make them more parallel?
Are we saying the cleansing should happen after being received? That sounds wrong...?

Reply:
I just deleted them altogether. Problem solved!